app/controllers/green_lanes/check_your_answers_controller.rb

```diff
@@ -11,6 +11,7 @@ def show
       @country_of_origin = check_your_answers_params[:country_of_origin] || GeographicalArea::ERGA_OMNES
       @moving_date = check_your_answers_params[:moving_date]
       @category_one_assessments = determine_category.cat1_with_exemptions
+      @category_two_assessments_without_exemptions = determine_category.cat2_without_exemptions.present?
```

Comment:
it seems ` @category_two_assessments_without_exemptions ` is an array, but actually it is a true/false value